dist/src/main/java/io/camunda/zeebe/shared/management/BackupEndpoint.java

```diff
@@ -109,6 +114,17 @@ public WebEndpointResponse<?> delete(@Selector @NonNull final long id) {
     }
   }
 
+  private WebEndpointResponse<?> state(final String type) {
+    try {
+      final var cpType =
+          BackupApi.BACKUP.equals(type) ? CheckpointType.SCHEDULED_BACKUP : CheckpointType.MARKER;
```

Comment:
🔧 I think it's surprising to users that they ask for the latest "backup" but then only see the latest scheduled backup.
I'd suggest that the actuator takes slightly different parameters: 
1. `latestBackup` -> returns the latest backup, manual or scheduled
2. `latestCheckpoint` -> returns the latest marker
And optionally we could have `latestManualBackup` and `latestScheduledBackup`, if we actually need it. For scheduling purposes, I think it doesn't matter whether the latest backup was a manual or scheduled one.


But also see my other comment, if we go with a parameter-less endpoint, we don't need to change this.

Reply:
Yeah that makes sense :+1: I guess it's just a bit confusing to see `SCHEDULED_BACKUP` used rather arbitrarily here to mean "actually, any backup type".